spec/services/outliers/calculate_distance_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "spec_helper"
+
+describe Outliers::CalculateDistance do
+  before(:each) do
+    @subject = Outliers::CalculateDistance.new
+  end
+
+  it "when the points are the same it returns 0" do
+    lng_lat = [one, one]
+
+    actual = @subject.call(lng_lat, lng_lat)
+
+    expect(actual).to eq(zero)
+  end
+
+  it "it returns the distance 1" do
+    lng_lat1 = [one, one]
+    lng_lat2 = [one, two]
+
+    actual = @subject.call(lng_lat1, lng_lat2)
+
+    expect(actual).to eq(one)
+  end
+
+  it "it returns the distance 2" do
```

Comment:
This test case returns the distance 1.41... not 2. Unless you just meant to enumerate the test cases this way?

Reply:
Second one. I was enumerating ;)